clang/test/CodeGen/aix-builtin-cpu-supports.c

```diff
@@ -0,0 +1,171 @@
+// RUN: echo "int main() { return __builtin_cpu_supports(\"4xxmac\");}" > %t.c
```

Comment:
This is an interesting way of testing, where we create each test on the fly. I am not against it if it works on all platforms (including Windows). However, I would like to see one test case where we have multiple calls to the builtins, with multiple uses of each "support method" in the same compilation unit.